symplyphysics/laws/relativistic/relativistic_momentum.py

```diff
@@ -0,0 +1,42 @@
+from sympy import (Eq, solve, sqrt)
+from sympy.physics.units import speed_of_light
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+)
+
+# Description
+## Momentum (amount of motion) is a vector physical quantity that is a measure of the mechanical movement
+## of a body. The relativistic angular momentum also takes into account speed limits equal to the speed of light.
+
+## Law is: p = m * v / sqrt(1 - v^2 / c^2), where
+## p - momentum,
+## m - mass,
+## v - velocity,
```

Comment:
Explain mass and velocity. Mass is relativistic or rest mass? Velocity is relativistic velocity? Relative to what?